tests/foreman/rhci/test_rhci.py

```diff
@@ -76,26 +78,25 @@ def test_positive_update_name(self, data):
         @Feature: RHCI Deployment
 
         """
-        deployment = entities.RHCIDeployment(
-            name=data['name'],
-            organization=self.org,
-            lifecycle_environment=self.lc_env,
-            deploy_rhev=True,
-            rhev_engine_host=self.rhev_engine_host,
-            rhev_storage_type='NFS',
-            rhev_engine_admin_password='changeme',
-        ).create()
-        self.assertEquals(deployment.name, data['name'])
+        for data in valid_name_update_tests():
```

Comment:
Suggesting removing that `valid_name_update_tests` and use same construction here as above:
```
    for new_name in (gen_string(str_type='alpha'),
                     gen_string(str_type='alphanumeric'),
                     gen_string(str_type='cjk'),
                     gen_string(str_type='latin1')):
```
We have already verified that entity can be created using any data type, so we can use `name=gen_string('alphanumeric') instead name=data['name']